birdhouse/components/weaver/config/magpie/config.yml.template

```diff
@@ -10,6 +10,31 @@ providers:
     c4i: false
     type: api   # FIXME: 'ades' when https://github.com/Ouranosinc/Magpie/issues/360 implemented
     sync_type: api
+    # hook locations should be relative to mounted Twitcher location as they are run within that container
+    # see following for hooks details:
+    # - https://github.com/Ouranosinc/Magpie/blob/master/config/providers.cfg
+    # - https://pavics-magpie.readthedocs.io/en/latest/configuration.html#service-hooks
+    hooks:
+      # when a job is created in weaver, apply the header that will nest output results under user's context directory
+      # see also:
+      # - https://pavics-weaver.readthedocs.io/en/latest/processes.html?highlight=x-wps-output-context#outputs-location
+      # each path below are equivalents, but with more or less specific reference to the requested service/process
+      - type: request
+        path: "/providers/[\\w_-]+/processes/[\\w_-]+/jobs"
+        method: POST
+        target: /opt/birdhouse/src/magpie/hooks/weaver_hooks.py:add_x_wps_output_context
+      - type: request
+        path: "/processes/[\\w_-]+/jobs"
+        method: POST
+        target: /opt/birdhouse/src/magpie/hooks/weaver_hooks.py:add_x_wps_output_context
+      - type: request
+        path: "/jobs"
```

Comment:
`path: "/jobs"` here will match to `https://PAVICS_HOST/ows/proxy/twitcher/jobs` ?

Reply:
It will match `https://PAVICS_HOST/ows/proxy/twitcher/ows/proxy/weaver/jobs`

Reply:
To clarify, the `path` in the context of Twitcher/Magpie hooks is whatever is defined after the resolved protected service. 
I also added a link in the python file to the reference doc in Magpie if you need more details.